src/create-embed.js

```diff
@@ -151,7 +152,14 @@ const wrapResponsive = (embedType, embedOptions, el) => {
   // This value is validate at a higher level, so we can trust that it's in the
   // correct format.
   const aspectRatio = responsive.aspectRatio.split(':').map(Number);
+
+  if (document.getElementsByClassName(DEAFULT_RESPONSIVE_CLASS).length) {
+    document.getElementsByClassName(DEAFULT_RESPONSIVE_CLASS)[0].remove();
+  }
```

Comment:
Are you sure the first responsive container is going to be an empty node and not one containing another player?